.github/workflows/push-example.yml

```diff
@@ -1,6 +1,5 @@
 on:
-  push:
-    branches: ['example-e2e']
+  pull_request:
```

Comment:
Maybe leave this on push, with a list of branches where we each have a target branch to test against.  This is essentially the flow to main/release/feature work branches